CMakeLists.txt

```diff
@@ -2,10 +2,15 @@ cmake_minimum_required(VERSION 3.14)
 
 project(zip
   LANGUAGES C
-  VERSION "0.3.0")
+  VERSION "2025.06.18")
```

Comment:
Why change the version scheme here? This doesn't simplify anything for vcpkg. It creates problems.